pkg/controller/statusmanager/status_manager_test.go

```diff
@@ -126,7 +126,7 @@ func (f *fakeRESTMapper) ResourceSingularizer(resource string) (singular string,
 }
 
 func TestStatusManager_set(t *testing.T) {
-	client := fake.NewFakeClient()
+	client := fake.NewClientBuilder().WithRuntimeObjects().Build()
```

Comment:
weird... did you copy this from somewhere? I would have expected `NewFakeClient()` to still work...

Reply:
when compiling a message prints from kube that `NewFakeClient()` is deprecated.

Reply:
ah. ok, but it looks like the `WithRuntimeObjects` part isn't needed if you aren't actually passing any objects. So just:

    client := fake.NewClientBuilder().Build()